lib/model/model.go

```diff
@@ -1526,14 +1523,16 @@ func (m *Model) localChangeDetected(folder, path string, files []protocol.FileIn
 			action = "deleted"
 		}
 
-		// The full file path, adjusted to the local path separator character.
-		path := filepath.Join(path, filepath.FromSlash(file.Name))
+		// The full file path, adjusted to the local path separator character.  Also
+		// for windows paths, strip unwanted chars from the front.
+		path := filepath.Join(strings.Replace(folderCfg.Path(), `\\?\`, "", 1), filepath.FromSlash(file.Name))
```

Comment:
Last comment, I think we should keep the `strings.Replace(folderCfg.Path(), `\\?\`, "", 1)` outside of the loop as it does not need to be done for each iteration and this stuff gets called a lot after all.